infer/src/dotnet/ResourceLeakCSDomain.ml

```diff
@@ -118,18 +118,27 @@ let assign lhs_access_path rhs_access_path held =
     | exception Caml.Not_found ->
         ()
   in
+  let compare_expr_base ((base1, _)) ((base2, _)) = 
+    if AccessPath.equal_base base1 base2 then true
+    else false
+  in
```

Comment:
minor: `compare_*` functions in OCaml typically return -1/0/1, this function should be called `equal_expr_base` or just `equal_base`. It can also be simplified:

```
let equal_base (base1, _) (base2, _) = AccessPath.equal_base base1 base2 in
```

Reply:
Done